crates/pop-parachains/src/new_parachain.rs

```diff
@@ -39,37 +35,50 @@ pub fn instantiate_standard_template(
 	config: Config,
 	tag_version: Option<String>,
 ) -> Result<Option<String>> {
-	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
-	let source = temp_dir.path();
+	// Template palceholder definitions
+	let mut token_symbol: String = "token-symbol=".to_string();
+	let mut token_decimals: String = "token-decimals=".to_string();
+	let mut initial_endowement: String = "initial-endowment=".to_string();
 
-	let tag = Git::clone_and_degit(template.repository_url()?, source, tag_version)?;
+	// Placeholder customization
+	token_symbol.push_str(&*config.symbol);
```

Comment:
Just small comment, this:
```
let mut token_symbol: String = "token-symbol=".to_string();
token_symbol.push_str(&*config.symbol);
```
can be refactor to:
```rs
let token_symbol: String = format!("token-symbol={}", config.symbol);
```

Reply:
Thanks, this makes more sense!